api/src/main/java/com/strategyobject/substrateclient/api/pallet/council/Council.java

```diff
@@ -0,0 +1,113 @@
+package com.strategyobject.substrateclient.api.pallet.council;
+
+
+import com.strategyobject.substrateclient.common.types.Result;
+import com.strategyobject.substrateclient.common.types.Unit;
+import com.strategyobject.substrateclient.pallet.annotation.Event;
+import com.strategyobject.substrateclient.pallet.annotation.Pallet;
+import com.strategyobject.substrateclient.rpc.api.AccountId;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash;
+import com.strategyobject.substrateclient.rpc.api.runtime.DispatchError;
+import com.strategyobject.substrateclient.scale.ScaleType;
+import com.strategyobject.substrateclient.scale.annotation.Scale;
+import com.strategyobject.substrateclient.scale.annotation.ScaleReader;
+import lombok.Getter;
+import lombok.Setter;
+
+@Pallet("Council")
+public interface Council {
+    /**
+     * A motion (given hash) has been proposed (by given account) with a threshold (given MemberCount).
+     */
+    @Event(index = 0)
+    @Getter
+    @Setter
+    @ScaleReader
+    class Proposed {
+        private AccountId account;
+        @Scale(ScaleType.U32.class)
+        private Long proposalIndex;
+        private Hash proposalHash;
+        @Scale(ScaleType.U32.class)
+        private Long threshold;
+    }
+
+    /**
+     *  A motion (given hash) has been voted on by given account, leaving
+     *  a tally (yes votes and no votes given respectively as `MemberCount`).
+     */
+    @Event(index = 1)
+    @Getter
+    @Setter
+    @ScaleReader
+    class Voted {
+        private AccountId account;
+        private Hash proposalHash;
+        private Boolean voted;
+        @Scale(ScaleType.U32.class)
+        private Long votedYes;
```

Comment:
Not sure if variable names matter, but `yes` and `no` is what's in the metadata rather than `votedYes` and `votedNo`